User Ratings/User Ratings/CustomControl.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  CustomControl.swift
+//  User Ratings
+//
+//  Created by Jonathan Ferrer on 5/21/19.
+//  Copyright © 2019 Jonathan Ferrer. All rights reserved.
+//
+
+import UIKit
+
+class CustomControl: UIControl {
+
+    var labels: [UILabel] = []
+    var value: Int = 1
+
+    private let componentDimension: CGFloat = 40.0
+    private let componentCount = 5
+    private let componentActiveColor: UIColor = .green
+    private let componentInactiveColor: UIColor = .gray
+
+    required init?(coder aCoder: NSCoder) {
+        super.init(coder: aCoder)
+        setup()
+    }
+
+    func setup() {
+        for label in 1...5 {
```

Comment:
I would say here ```for label in 1...componentCount```
that way if you ever modify the components count you don't have to change it in multiple places and its the count of all labels.